kinto_client/cli_utils.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import getpass
+import logging
+
+from . import Client
+
+
+def get_auth(auth):
+    """Ask for the user password if needed."""
+    auth = tuple(auth.split(':', 1))
+    if len(auth) < 2:
+        user = auth[0]
+        password = getpass.getpass('Please enter a password for %s: '
+                                   % user)
+        auth = (user, password)
+
+    return auth
+
+
+def client_from_args(args):
+    """Return a client from parser args."""
+    return Client(server_url=args.server,
+                  auth=get_auth(args.auth),
```

Comment:
We should probably call get_auth at the same time we get the options. Not sure if it's possible to do it via argparse?